Added Edmonton Spends for 2024 FY

5ah opened the pull request.

5ah commented:
The page reports the budget and actual side by side. I pulled the data from the '2024' column which represents the actual revenue/spend for the fiscal year, whereas the numbers you noted match the 'Budget' column.

The reason for the school taxes 'drop off' is because the city collects taxes for the provincial government so it looks like its dropping off. Should I leave it as is or add a destination node labeled 'Transfer to Province' to show the money flow.

xrendan commented:
Can you add a destination node for the school taxes?

xrendan commented:
You're right on the protective services number, I'm not sure what I was doing there. 

The government transfer is still weird. 

The chart says 701.5M but the financial accounts say 165M

5ah commented:
### 1. Visualization & Data Updates

I have updated the Sankey diagram to clarify the flows for Taxes and Government Transfers.

**A. Taxes (Transfers to Partners)** 
I created a new destination node under Spending labeled Transfers to partners to handle money the City collects but passes on to third parties. This fixes the visual "drop off."

- Education: Transferred to the Provincial Government.

- Business Improvement Areas: Transferred to Local Business Boards.

**B. Government Transfers (Operating vs. Capital)** 
I have separated the Government Transfers node into Operating and Capital to explicitly show the difference in usage restrictions.

1. Operating Transfers: Funding for day-to-day services (social services, housing support, policing).

- > Example: $23.3M for Family and Community Support Services (social workers/programs).

- > Example: $43.6M for Debt Servicing (paying interest on projects like the LRT).

2. Capital Transfers: Restricted funding for building/buying physical infrastructure (bridges, roads, transit). This money cannot be used for salaries or daily costs.

- > Example: $133M from LGFF (Provincial grant for infrastructure).

- > Example: $58.6M for Terwillegar Drive & Yellowhead Trail upgrades.

- > Example: $33.6M for Valley Line LRT construction.

_Note: I pulled these specific examples from `Note 21` in the 2024 Financial Statements. A full 100% breakdown isn't possible as the report does not have the data._

### 2. Feature Update: Multiple Sources (#196)

I have also addressed issue #196 by adding support for multiple source citations.

Changes:

- `jurisdictions.tsx:` Added a sources array to the interface to handle multiple entries with scope.
```
sources?: Array<{
  label: string;
  url: string;
  scope?: string;
}>;
```

- `pages.tsx:` Updated the logic to check for sources first; falls back to the original source string if the array is missing.
Example Usage `(summary.json)`:
```
  "sources": [
    {
      "label": "Financial Annual Report Consolidated Financial Statements 2024",
      "url": "https://www.edmonton.ca/sites/default/files/public-files/FinancialAnnualReportConsolidatedFinancialStatements2024.pdf?cb=1761586265",
      "scope": "City-level revenue and spending"
    },
    {
      "label": "Financial Annual Report 2024",
      "url": "https://www.edmonton.ca/sites/default/files/public-files/2024FinancialAnnualReport.pdf?cb=1746489478",
      "scope": "Debt and employee data"
    }
  ],
```

xrendan reviewed (approved):
This looks good to me. 

We really have to figure out a way to show that accumulated surplus ends up being spent and is accounted for in capital assets that the government holds.
